### ISSUE TITLE
可以问下这句怎么理解嘛

### ISSUE THREAD

**Issue by upupbo** [lidq92/MDTVSFA#11]
m = train_dataset.m
m代表什么，为什么可以train_dataset后面直接加.m

**Comment by lidq92** [comment]
VQADataset class有定义m这个属性，看代码Line15&24就行。
m代表数据集里主观分数的最小值，另一个scale代表主观分数范围。二者用来初始化alignment的fc中的参数，以及scale用来归一化regression loss。
